Kiosk/SettingsHelper.cs

```diff
@@ -276,6 +277,29 @@ private async void LoadRoamingSettings()
                 this.TranslatorTextApiKey = value.ToString();
             }
 
+            value = ApplicationData.Current.RoamingSettings.Values["InkRecognizerApiKey"];
```

Comment:
See what we did for the Translator Text case. I believe we just need a setting for the api key, which will greatly simply the changes to this file.

Reply:
The only option available now in the settings page is to set an API key.